examples/loader-google-bigquery/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf src/.observablehq/cache",
+    "build": "rimraf dist && observable build",
+    "dev": "observable preview",
+    "deploy": "observable deploy",
+    "observable": "observable"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery": "^7.8.0",
+    "@observablehq/framework": "^1.8.0",
```

Comment:
this could use the current version of framework, or "latest"

Reply:
This is fine as-is. I don’t think we shops recommend using latest over a tilde or caret version range which is more common and safer. Even though this doesn’t refer to the latest version of Framework, you will still get the latest version that is compatible with this range when you install, and this is the same pattern that is used by the other examples and we want to be consistent.